shared-files/spotbugs-excludes.xml

```diff
@@ -0,0 +1,7 @@
+<FindBugsFilter>
+    <Match>
+        <Class name="*"/> <!-- You can specify a specific class name if needed -->
+        <Method name="*"/> <!-- You can specify a specific method name if needed -->
+        <Bug code="EI_EXPOSE_REP2"/>
```

Comment:
@devtobi we probably should discuss about this.
In my opinion there is no other way, as often it is cleaner that way and it travels up the call tree if suppressed in one spot.

Reply:
Reverted and done with Annotation.
Will be discussed for Spotbugs with integrations.